vendor/init.bat

```diff
@@ -160,7 +160,7 @@ if not defined HOME set "HOME=%USERPROFILE%"
 :: This is either a env variable set by the user or the result of
 :: cmder.exe setting this variable due to a commandline argument or a "cmder here"
 if defined CMDER_START (
-    cd /d "%CMDER_START%"
+    cd /d "%CMDER_START%\"
```

Comment:
Don't assume you need to add a '\', instead add the below between 162 and 163:

if not "%CMDER_START:~-1%" == "\" SET "CMDER_START=%CMDER_START%\"

Then remove the '\' from line 163